mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job.go.erb

```diff
@@ -364,7 +364,16 @@ func resourceDataflowFlexJobSetupEnv(d *schema.ResourceData, config *transport_t
 	var enableStreamingEngine bool
 	if p, ok := d.GetOk("parameters.enableStreamingEngine"); ok {
 		delete(updatedParameters, "enableStreamingEngine")
-		enableStreamingEngine = p.(bool)
+		e := strings.ToLower(p.(string))
```

Comment:
My understanding of the original bug is that Map type fields are stored as string:string maps in state, so even if the value in the config is a boolean true/false it will be converted to "true"/"false" in state.